discordBot.js

```diff
@@ -29,32 +29,82 @@ class DiscordBot {
      return client.login(this.token);
   }
 
-  sendMessageToAllChannels(msg) {
-    const sendAll = this.client.channels.cache
-      .filter(ch => ch.type == 'text')
-      .map(ch => ch.send(msg));
+  async sendMessageToAllChannels(msg) {
+    const channels = await this.fetchChannelsInternal();
+    const sendAll = channels.map(ch => ch.send(msg));
 
     return Promise.all(sendAll);
   }
 
-  sendMessageChannel(channelId, message) {
-    return this.client.channels.cache
-      .get(channelId)
-      .send(message);
+  async sendMessageChannel(channelId, message) {
+    const channel = await this.client.channels.fetch(channelId)
+    return channel.send(message);
   }
```

Comment:
same thing as send file. send doesnt work unless inside this.client.on('ready'...

Reply:
disregard i was running it wrong lol